community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/LoadCsvAcceptanceTest.scala

```diff
@@ -381,6 +381,20 @@ class LoadCsvAcceptanceTest
     result.toList should equal(List(Map("field" -> "something")))
   }
 
+  test("should restrict file urls to be rooted within an authorized directory") {
+    val dir = createTempDirectory("loadcsvroot")
+
+    val db = new TestGraphDatabaseFactory()
```

Comment:
@pontusmelke I guess, but I don't know why. Is this something you think should be done before merging this?

This is how the rest of the tests are written in this file. I simply copy pasted another test in 3.0, then in 2.3 I modified it (by copying from tests there) to work in that API. Paired a bit with @systay on it.